docs/RUNNING_ON_A_BUDGET.md

```
@@ -65,6 +65,78 @@ To configure OpenCode with a custom provider:
    $env:OPENAI_API_BASE="https://openrouter.ai/api/v1"
    $env:OPENAI_API_KEY="your_openrouter_api_key_here"
    ```
+### Kimi K2.5 via OpenCode (Verified)
```

markdownlint-cli2 (0.23.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Above

(MD022, blanks-around-headings)